orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -115,7 +115,8 @@ def load_features(self):
         self.selected_features = []
         if self.corpus is not None:
             self.features = [(meta.name, vartype(meta)) for meta in self.corpus.domain.metas] + \
-                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes]
+                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes] + \
+                            [(class_var.name, vartype(class_var)) for class_var in self.corpus.domain.class_vars]
```

Comment:
I wouldn't bet my pinky on what it returns without looking.

Why doesn't it iterate over the contents of the domain, namely including meta attributes? I know the answer is _legacy_, but semantically, metas are as part of our tables as anything (cf. pandas).